added TypeScript Badges

ellerbrock opened the pull request.

dzharii commented:
Hi @ellerbrock 

Thank you for your contribution, but for now I doubt I can accept this pull request. 
The reasons are: 

1. The image reference `aka <img src="..."> in html` to an external website `https://badges.frapsoft.com/....png` might be a hidden user activity 
tracking script. I would like to avoid the risks of *hidden tracking* and I propose to include this image assets into this pull request and than make a local image reference inside the readme.md.

2. I would suggest to move the badges at the bottom of the readme.md file before or after the "Social" section:
https://github.com/dzharii/awesome-typescript#social
You could create a new section for "Badges" as well. 

Kind regards, 
Dmytro @dzharii

ellerbrock commented:
hi @dzharii,

i changed the src to the raw image files inside the repository (https://raw.githubusercontent.com), hope thats alright. i guess image tracking should be not possible on github anymore since github is caching the images and serving them over a proxy solution looks like this if you look for some pictures  https://camo.githubusercontent.com/e319a9f96be933a9b8cc12e9014b1a...
and sorry that i made a commit just before, i updated the file via web interface and hit enter to fast :)

cheers maik

dzharii commented:
Cool! Thank you @ellerbrock!

Merged.
